modules/setting/admin.go

```diff
@@ -19,6 +20,7 @@ func loadAdminFrom(rootCfg ConfigProvider) {
 	sec := rootCfg.Section("admin")
 	Admin.DisableRegularOrgCreation = sec.Key("DISABLE_REGULAR_ORG_CREATION").MustBool(false)
 	Admin.DefaultEmailNotification = sec.Key("DEFAULT_EMAIL_NOTIFICATIONS").MustString("enabled")
+	Admin.DefaultUINotification = sec.Key("DEFAULT_UI_NOTIFICATIONS").MustString("enabled")
```

Comment:
Also add the description in `app.example.ini` and `docs` like `DEFAULT_EMAIL_NOTIFICATIONS`